plugins/vpp/l3plugin/vppcalls/vpp2001/dhcpproxy_vppcalls.go

```diff
@@ -50,9 +50,11 @@ func (h *DHCPProxyHandler) createDeleteDHCPProxy(entry *l3.DHCPProxy, delete boo
 		}
 
 		if ipAddr.To4() == nil {
-			config.DHCPServer = []byte(ipAddr.To16())
+			copy(config.DHCPServer.Un.XXX_UnionData[:], ipAddr.To16())
```

Comment:
Can you make a function for converting `net.IP` to `dhcp.Address` like it's done in other plugins. The `XXX_` fields should not really be modified directly, as those for govpp internal use and using them in many places creates additional work when they get removed in the future.